cleanup (#595)
SUSE/doc-susemanager: modules/administration/pages/public-cloud.adoc

A world-accessible {productname} instance violates the terms of the {productname} EULA, and it will not be supported by {suse}.
====

> To be able accessing the {productname} {webui} allow HTTPS, when you are setting up your networking environment.

Reviewer: ```suggestion
To access the {productname} {webui}, allow HTTPS when you set up your networking environment.
```
